.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,38 +1,20 @@
-<!--
-Copyright 2024 Ocean Protocol Foundation
-SPDX-License-Identifier: Apache-2.0
--->
----
-name: Enhancement
-about: 'Benefit_yyy, via building_xxx'
-title: '[Component name] Benefit_yyy, via building_xxx'
-labels: 'Type: Enhancement'
-assignees: ''
-
----
+Title: 
```

Comment:
Please use the format "name:", "about:" etc so that it can properly fill in the various fields.